manifests/init.pp

```diff
@@ -286,6 +292,9 @@
   Optional[Redis::RedisUrl] $jobs_sidekiq_redis_url = $::foreman::params::jobs_sidekiq_redis_url,
   Boolean $hsts_enabled = $::foreman::params::hsts_enabled,
   Array[Stdlib::HTTPUrl] $cors_domains = $::foreman::params::cors_domains,
+  Integer[0] $foreman_puma_threads_min = $::foreman::params::foreman_puma_threads_min,
```

Comment:
I'd group these together with the other service options to at least keep some logic to it even though it probably doesn't matter that much.

Reply:
Do you mean `foreman_service`, `foreman_service_ensure`, `foreman_service_enable`, `foreman_service_port`? I don't see them in init.pp. Should I add those?